homeassistant/components/homeassistant/strings.json

```diff
@@ -10,6 +10,10 @@
       "title": "The country has not been configured",
       "description": "No country has been configured, please update the configuration by clicking on the \"learn more\" button below."
     },
+    "imperial_unit_system": {
+      "title": "The imperial unit system is deprecated",
+      "description": "The imperial unit system is deprecated and your system is currently using us customary. Please update your configuration to use the us customary unit system and reload the core configuration to fix this issue."
```

Comment:
Should this be US in capital letters?

Reply:
Actually since this is a yaml only issue, maybe we should use `us_customary` instead to refer to the yaml value (and put it in placeholders to avoid translations of it.